ACE/ace/config-linux-common.h

```diff
@@ -173,7 +173,7 @@
 
 // Starting with FC9 rawhide this file is not available anymore but
 // this define is set
-#if defined _XOPEN_STREAMS && _XOPEN_STREAMS == -1
+#if (defined(__GLIBC__) && (__GLIBC__ == 2 && __GLIBC_MINOR__ >= 30)) || (defined _XOPEN_STREAMS && _XOPEN_STREAMS == -1)
```

Comment:
What when glibc 3.x gets released?

Reply:
Is _XOPEN_STREAMS defined with glibc v2.30? If not, maybe check !_XOPEN_STREAMS?

Reply:
I like that solution much better than glibc specific check. Regardless, @saininav do you know why glibc dropped this macro? If we could pass macros to glibc to get it back, it would be an even better solution.

Reply:
From Glibc 2.30 News File:

> * The obsolete and never-implemented XSI STREAMS header files <stropts.h> and <sys/stropts.h> have been removed.

Nevermind about getting it back I guess. However if it was never implemented does that mean this never actually worked with glibc?

Reply:
I also think so, as _XOPEN_STREAMS == -1 was true in previous versions.